pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -79,12 +81,16 @@ def addRunSteps(outFile, runTargets):
 
 def addBuildConfig(outFile, num, name, buildArgs, buildMeFile):
     outFile.write('<valuemap type="QVariantMap" key="ProjectExplorer.Target.BuildConfiguration.' + str(num) + '">\n')
-    outFile.write('<value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory">' + os.getcwd() + '</value>\n')
+    outFile.write('<value type="QString" key="ProjectExplorer.BuildConfiguration.BuildDirectory">' + (os.getcwd() if not isWindows() else os.popen('pwd -W').read().replace('\n', '')) + '</value>\n')
     outFile.write('<valuemap type="QVariantMap" key="ProjectExplorer.BuildConfiguration.BuildStepList.0">\n')
     outFile.write(' <valuemap type="QVariantMap" key="ProjectExplorer.BuildStepList.Step.0">\n')
     outFile.write('  <value type="bool" key="ProjectExplorer.BuildStep.Enabled">true</value>\n')
-    outFile.write('  <value type="QString" key="ProjectExplorer.ProcessStep.Arguments">' + buildArgs + '</value>\n')
-    outFile.write('  <value type="QString" key="ProjectExplorer.ProcessStep.Command">' + buildMeFile + '</value>\n')
+    if isWindows():
+        outFile.write('  <value type="QString" key="ProjectExplorer.ProcessStep.Arguments">-msys2 -defterm -no-start -use-full-path -here -c "PATH=~/bob:/usr/bin /usr/bin/sh ' + buildMeFile + ' -v ' + buildArgs + '"</value>\n')
+        outFile.write('  <value type="QString" key="ProjectExplorer.ProcessStep.Command">' + os.path.normpath(os.path.join(os.getenv('WD').replace('\\', '/'), '..', '..', 'msys2_shell.cmd')) + '</value>\n')
```

Comment:
It is imported to start msys2 by "msys2_shell.cmd" to get WD environment-vaiable.
I got no better solution to get install path of msys2 to execute bob by qt-creator.

Reply:
Looking at `PATH`: does Bob has to be installed in `~/bob`? Could that be taken from `os.getenv('PATH')` instead?